### PR TITLE
DVCSMP-2104 - Harmony - Fix Exceptions

### PR DESCRIPTION
**Pull request by juano2310**
- MissingPropertyException: No such property: code for class: java.lang.String
- NullPointerException: Cannot get property 'activities' on null object
- NullPointerException: Cannot get property 'device' on null object

### CODE REVIEW

**Comment by juano2310** [comment]
@jimmyjames @DougSven please take a look. This should solve most of the biggest exceptions for this Smartapp. Thanks!

File: smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy [hunk 1]
```diff
@@ -557,58 +550,6 @@ def pollResponse(response, data) {
 	}
 }
 
-def getActivityList() {
```
Comment (DougSven) [inline smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy:560]: It's not necessary to remove this method unless it's no longer needed. The method missing exceptions were related to executing an old version of the installed smart app

Comment (juano2310) [reply smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy:560]: Is not longer needed. It was updating a list that was not being use.

Comment (DougSven) [reply smartapps/smartthings/logitech-harmony-connect.src/logitech-harmony-connect.groovy:560]: right on

**Comment by DougSven** [comment]
👍 Looks good.

**Comment by workingmonk** [comment]
please add the ticket number to the PR title

**Pull request merged**